addons/rose/addon/components/rose/form/checkbox/index.hbs

```diff
@@ -4,7 +4,7 @@
 }}
 
 <div class='rose-form-checkbox{{if @error " error"}} {{if @inline " inline"}}'>
-
+  {{! template-lint-disable no-builtin-form-components }}
```

Comment:
Disabling since this `Rose::Form::Checkbox` component is only used in rose dropdowns and that is being refactored to use hds already.

Reply:
HDS to da powah!!! 🙌